index.html

```diff
@@ -80,8 +80,8 @@
 						<div class="row hide">
 							<label for="gasPrice"><input type="number" id="gasPrice" value="0"> Gas Price</label>
 						</div>
-						<div class="row hide">
-							<label for="value"><input type="number" id="value" value="0"> Value</label>
+						<div class="row">
+							<label for="value"><input type="text" id="value" value="0"> Value</label>
```

Comment:
Please make it clear that this defaults to ether and other units can be used, i.e.
` Value (e.g. .7 ether or 5 wei, defaults to ether)`